arch-travis.sh

```diff
@@ -192,18 +192,24 @@ run_build_script() {
 
 # setup pacaur
 setup_pacaur() {
-  local cowerarchive="cower.tar.gz"
-  local aururl="https://aur.archlinux.org/cgit/aur.git/snapshot/"
-  # install cower
-  as_normal "curl -O $aururl/$cowerarchive"
-  as_normal "tar xf $cowerarchive"
-  chroot_as_normal "cd cower && makepkg -is --skippgpcheck --noconfirm"
-  as_root "rm -r cower"
-  as_normal "rm $cowerarchive"
-  # install pacaur
-  chroot_as_normal "cower -dd pacaur"
-  chroot_as_normal "cd pacaur && makepkg -is --noconfirm"
-  chroot_as_normal "rm -rf pacaur"
+  # Check if pacaur is available in the added repos
+  local pacaur=$(chroot_as_normal "pacman -Si pacaur &> /dev/null || echo '0'")
+  if [[ -z $pacaur ]]; then
```

Comment:
I have just pushed a new commit that makes something like this more clean.

Please rebase against 41012ab7 and use:

```bash
if _chroot_as_normal "pacman -Si pacaur &> /dev/null"; then
```
instead of the above. Then I will merge it :)